### DISCUSSION TITLE
How to change table hover color?

### DISCUSSION THREAD

**Discussion by arcanechimp** [saadeghi/daisyui#1485] [Q&A] [answered]
What would be the most idiomatic way of changing the default hover color on a table row?

**Comment by saadeghi** [comment, marked as answer]
To edit this or any other thing you can add custom CSS to your CSS file. (By inspecting the element using dev tools, copy the selector rule that affecting your element then customize it and add the new CSS to your CSS file.)

- In this case you can also do it by adding `[&_tr.hover:hover_*]:!bg-teal-400` class name to `table` this selects the cells in a hovered row and applies `bg-teal-400` color to it.
- You can also customize the color value itself. The hovered row is using [`--b3` (base-300)](https://daisyui.com/docs/colors/) color for hover so you can change `--b3` color to something else by adding a class name like this `[--b3:172_66%_50%]` to the `table` (those numbers are HSL color values)

Example: https://play.tailwindcss.com/9QsUGLZfO1

**Reply by arcanechimp** [reply]
Thank you! Much appreciated!
